.github/workflows/docker.yml

```diff
@@ -21,8 +21,14 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: trdo/tezos-reward-distributor
       - name: Push to Docker Hub
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v5
```

Comment:
Does docker/build-push-action has already a version 5? On their [github page](https://github.com/docker/build-push-action) they still use v2 (see [example](https://github.com/docker/build-push-action/blob/master/docs/advanced/tags-labels.md)). Or am I missing something?

Reply:
My bad, I was looking at the wrong action.